terraform-ss-proxy/main.tf

```diff
@@ -0,0 +1,224 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
```

Comment:
Can this be pinned more specifically? I'm just worried about the edge-case of some _other_ terraform code trying to interact with resources created by this choking on a version difference (and yes, that happens, and is why [unity-proxy is very specifically pinned](https://github.com/unity-sds/unity-proxy/blob/main/terraform-unity/versions.tf#L8-L11))